feel/feeldiscr/operatorinterpolation.hpp

```diff
@@ -1074,30 +1258,26 @@ OperatorInterpolation<DomainSpaceType, ImageSpaceType,IteratorRange,InterpType>:
 
                                 for ( uint16_type jloc = 0; jloc < domain_basis_type::nLocalDof; ++jloc )
                                 {
-                                    uint16_type compDomain = (domain_basis_type::is_product)? comp : 0;
+                                    uint16_type nCompDomain = (domain_basis_type::is_product)?domain_basis_type::nComponents:1;
+                                    for(int cdomain = 0;cdomain < nCompDomain;++cdomain )
+                                    {
```

Comment:
imagine for example an expression to interpolate that uses the cross product, then you would have a mix of component, or the curl for that matter.
But this will be for another time, I will, for Lagrange spaces (domain and image both) that we suppose that the components are not coupled